### ISSUE TITLE
Whatsapp: Couldn't open link on android

### ISSUE THREAD

**Issue by stuartcusackie** [ellisonleao/sharer.js#222]
Whatsapp sharing seems to work fine on desktop, but on android I get:
`"Couldn't open link".`

Here's my markup:

```
<button data-sharer="whatsapp" data-title="Test" data-url="https://ellisonleao.github.io/sharer.js/" data-link="true" data-blank="true">
ICON
</button>
```

As you can see I am testing with the sample url in an attempt to fix the problem, but that doesn't work.

**Comment by ellisonleao** [comment]
@stuartcusackie just pushed a fix for 0.5.1 and tested on my android device and it looks okay now.

**Comment by stuartcusackie** [comment]
Works perfectly now. Thank you very much!
